app/routes/send-endringsmelding.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import TekstBlokk from '~/komponenter/tekstblokk/TekstBlokk';
+import { ESanitySteg } from '~/typer/sanity/sanity';
+import { TypografiTyper } from '~/typer/typografi';
+import { useTekster } from '~/hooks/contextHooks';
+import InnholdKonteiner from '~/komponenter/innholdkonteiner/InnholdKonteiner';
 
 export default function SendEndringsmelding() {
-  return <p>Send Endringsmelding</p>;
+  const tekster = useTekster(ESanitySteg.SEND_ENDRINGER);
+
+  return (
+    <InnholdKonteiner>
```

Comment:
Burde denne bli brukt for `_index.tsx` også?

Reply:
Se commit cef072f ☀️

Reply:
🌟